.github/workflows/desktop-build.yml

```diff
@@ -271,16 +282,30 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v5
+        with:
+          submodules: recursive
 
-      - name: Install dependencies using Homebrew
-        shell: bash
-        # CMake cannot find the MySQL connector
-        # Neither of these works: mariadb-connector-c mysql-connector-c++
-        env:
-          HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
-        run: |
-          brew update
-          brew install ccache protobuf qt --force-bottle
+      - name: Setup Ccache
```

Comment:
TODO(this pr): add a comment about why this specific action instead of brew or any other similar action